deps/exokit-bindings/canvascontext/src/canvas-context.cc

```diff
@@ -264,38 +310,36 @@ float getFontBaseline(const SkPaint &paint, const TextBaseline &textBaseline, fl
   // If you changed the heuristic here, for consistency please also change it
   // in SimpleFontData::platformInit().
   // TODO(fserb): revisit this.
-  switch (textBaseline) {
-    case TextBaseline::TOP:
-      return fontMetrics.fAscent;
-    case TextBaseline::HANGING:
-      // According to
-      // http://wiki.apache.org/xmlgraphics-fop/LineLayout/AlignmentHandling
-      // "FOP (Formatting Objects Processor) puts the hanging baseline at 80% of
-      // the ascender height"
-      return fontMetrics.fAscent * 80.0f / 100.0f;
-    case TextBaseline::BOTTOM:
-    case TextBaseline::IDEOGRAPHIC:
-      return -fontMetrics.fDescent;
-    case TextBaseline::MIDDLE:
-      return -fontMetrics.fDescent + fontMetrics.fCapHeight * lineHeight / 2.0f;
-    case TextBaseline::ALPHABETIC:
-    default:
-      // Do nothing.
-      break;
+  if (textBaseline == kTextBaseline_TOP) {
```

Comment:
How was this change derived? Since it's quite different from the (possibly wrong) code before.

Reply:
I tried to match http://tutorials.jenkov.com/html5-canvas/text.html

![image](https://user-images.githubusercontent.com/59632/56109572-ccb76100-5f16-11e9-95f2-e0eb99e3b7f3.png)

Reply:
Does this PR match up with that render?